manifests/config.pp

```diff
@@ -32,7 +39,6 @@
     [ '/etc/caddy/config' ]:
       purge   => true,
       recurse => true,
-      require => User[$caddy::caddy_user],
     ;
   }
```

Comment:
this entire block reads strange, because of the `;`
you don't see that very often

Reply:
Yes, but I am following the official documentation. This will probably be changed by one of the next pull requests.